docs/nodes/widgets/ui-control.md

```diff
@@ -1,5 +1,6 @@
 ---
 {
+    "description": "Master the ui-control widget in Node-RED Dashboard 2.0 to manage dashboard states and data flows.",
```

Comment:
The ui-control is more about dynamically updating the elements from node-red side. Not sure this description fits.

Reply:
```suggestion
    "description": "Dynamically control elements on your dashboard through ui-control for Node-RED Dashboard 2.0.",
```